src/components/05_pages/NodeAddForm/index.js

```diff
@@ -38,19 +46,27 @@ class NodeAddForm extends React.Component {
   };
 
   render() {
-    return (
-      this.props.schema && (
+    let result = null;
+    if (this.props.schema) {
+      const entity = createEntity(this.props.schema);
```

Comment:
How about moving all this into a helper method to make it more readable?

Reply:
👍